### DISCUSSION TITLE
DSQD (720k 5.25" floppy) IMD files unrecognized.

### DISCUSSION THREAD

**Discussion by zombienerd** [keirf/disk-utilities#98] [Q&A]
Trying to use disk-utilities to convert a 720k DSQD 5.25" image to a binary/raw image, and I get the following:

disk-analyse: IMD: Unknown track mode/density 0x15
disk-analyse: 2000DOS.IMD: Bad disk image
disk-analyse: Failed to probe input file: 2000DOS.IMD

This is an IMD file that works as expected when re-written to disk, or converted to hfe and used in HxC or FlashFloppy emulator.

**Comment by keirf** [comment]
Probably just a new media identifier needs to be recognised by disk utilities. Please raise an issue on github instead of discussion topic as this is a bug to be fixed.
